extensions/data-plane/data-plane-api/src/main/java/org/eclipse/edc/connector/dataplane/api/DataPlaneApiExtension.java

```diff
@@ -67,6 +65,9 @@ public class DataPlaneApiExtension implements ServiceExtension {
     @Inject
     private DataPlaneManager dataPlaneManager;
```

Comment:
Can be removed?

Reply:
no, because it is used by `DataPlaneControlApiController`